mriqc/utils/__init__.py

```diff
@@ -1,8 +1,8 @@
 # emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
 # vi: set ft=python sts=4 ts=4 sw=4 et:
 """Module utils.misc contains utilities."""
-from .misc import reorder_csv
-from .bids import collect_bids_data
+from mriqc.utils.bids import collect_bids_data
+from mriqc.utils.misc import reorder_csv
```

Comment:
```suggestion
from .bids import collect_bids_data
from .misc import reorder_csv
```
Why absolute imports here?

Reply:
[PEP 8](https://pep8.org/#imports):

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path):
> 
> ```Python
> import mypkg.sibling
> from mypkg import sibling
> from mypkg.sibling import example
> ```
> 
> However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose:
> 
> ```Python
> from . import sibling
> from .sibling import example
> ```
> 
> Standard library code should avoid complex package layouts and always use absolute imports.
> 
> Implicit relative imports should never be used and have been removed in Python 3.

On this advice, I've generally been moving toward absolute imports when I update code.

Reply:
Hmmm, that would mandate to run unit tests after installing or explicitly manipulating the PYTHONPATH, correct?

I'm not convinced it is more readable and given that the recommendation is pretty soft, I honestly would rather keep the relative imports.

On the plus side, relative imports help black not to reorder the imports wrongly, alphabetically -- which is another PEP8 recommendation.

Reply:
> Hmmm, that would mandate to run unit tests after installing or explicitly manipulating the PYTHONPATH, correct?

If I understand correctly then no, it shouldn't require that, but in any case let me know whatever you decide, of course I have no problem changing it back if you think it's better.

Reply:
`autodoc` might require adding `sys.path.insert(0, os.path.abspath("../../"))`, but considering the documentation seems to be building successfully, it doesn't seem to be an issue.